requirements.txt

```diff
@@ -1,3 +1,3 @@
-bs4~=0.0.1
-requests~=2.11.1
+beautifulsoup4~=4.5.1
```

Comment:
I don't think so. Everything works fine and bs4 just downloads Beautiful Soup 4.. Also pytest passes. So...